requirements-dev.txt

```diff
@@ -0,0 +1,10 @@
+pre-commit==2.13.0
```

Comment:
This file can include `-r requirements.txt` on the first line to install all dev and core dependencies, so step 3 on the install can just be `pip install -r requirements-dev.txt`

Reply:
Wow! I didn't know about that, awesome!